contracts/Comptroller/Diamond/Diamond.sol

```diff
@@ -0,0 +1,200 @@
+pragma solidity 0.5.16;
+pragma experimental ABIEncoderV2;
+
+import { IDiamondCut } from "./interfaces/IDiamondCut.sol";
+import "../ComptrollerStorage.sol";
+import "../Unitroller.sol";
+
+contract Diamond is ComptrollerV12Storage {
```

Comment:
Should it extend also `ComptrollerInterfaceG2`? What will happen with the constant `isComptroller`?

Reply:
I think no need for an interface here as it's a proxy contract.